sdk/src/main/java/ch/admin/bag/covidcertificate/eval/utils/RecoveryEntryUtils.kt

```diff
@@ -11,14 +11,15 @@
 package ch.admin.bag.covidcertificate.eval.utils
 
 import ch.admin.bag.covidcertificate.eval.euhealthcert.RecoveryEntry
+import ch.admin.bag.covidcertificate.eval.models.AcceptanceCriterias
 import java.time.LocalDate
 import java.time.LocalDateTime
 import java.time.format.DateTimeFormatter.ISO_DATE
 import java.util.*
 
 
 fun RecoveryEntry.isTargetDiseaseCorrect(): Boolean {
-	return this.disease == AcceptanceCriterias.TARGET_DISEASE
+	return this.disease == ch.admin.bag.covidcertificate.eval.utils.AcceptanceCriterias.TARGET_DISEASE
```

Comment:
At this point it would be good if we could rename the utils object to something like `AcceptanceCriteriasConstants` (also to make it less confusing elsewhere - you don't always want to look at the import to figure out which one it is).

The same applied to the Test- and VaccinationEntryUtils.

Reply:
Changed Name